app/models/recurring_todos/abstract_repeat_pattern.rb

```diff
@@ -174,7 +174,7 @@ def continues_recurring?(previous)
     # same day as the previous
     def determine_start(previous, offset=0.day)
       start = self.start_from || NullTime.new
-      now = Time.zone.now
+      now = Time.now
```

Comment:
Regardless of what the app tries to save in the database, ActiveRecord always converts to UTC before storing it, and will convert back on retrieval. Tracks should not need any of its own logic to handle this correctly.